crates/engine/tree/src/tree/payload_processor/prewarm.rs

```diff
@@ -253,35 +213,47 @@ where
         Some((evm, evm_config, metrics))
     }
 
-    /// Transacts the transaction and returns the state outcome.
+    /// Transacts the vec of transactions and returns the state outcome.
     ///
-    /// Returns `None` if executing the transaction failed to a non Revert error.
+    /// Returns `None` if executing the transactions failed to a non Revert error.
     /// Returns the touched+modified state of the transaction.
     ///
-    /// Note: Since here are no ordering guarantees this won't the state the tx produces when
+    /// Note: Since here are no ordering guarantees this won't the state the txs produce when
     /// executed sequentially.
-    fn transact(self, tx: Recovered<N::SignedTx>) -> Option<EvmState> {
+    fn transact_batch(
+        self,
+        txs: &[Recovered<N::SignedTx>],
+        sender: Sender<PrewarmTaskEvent>,
+    ) -> Option<()> {
```

Comment:
looks like we can remove the return value here entirely?

Reply:
sure done here https://github.com/paradigmxyz/reth/pull/15155/commits/14da12d8f67f1ef67dc379dfdfd0735caaf562f7